draft-ietf-rats-corim.md

```diff
@@ -453,6 +453,8 @@ that MUST have a different identifier.
 {::include cddl/profile-type-choice.cddl}
 ~~~
 
+A "standard CoRIM type" references definitions in {{sec-corim-cddl}} and any further definitions in relevant IANA registries.
```

Comment:
```suggestion
A "standard CoRIM type" references definitions in {{sec-corim-cddl}} and any further definitions in relevant standards or IANA registries.
```

Reply:
Is this assumed to be IETF standard, or is there a list of expected standards bodies that extensions could come from?

Reply:
There could be other I-D from IETF or even other standards organizations could extend. There are some control surfaces in terms of IANA registries for code points and CBOR tagging, but IANA doesn't constrain itself to IETF-only standards. 

I'm not entirely clear on why this kind of wording in needed in the first place however. Is the "standard CoRIM type" terminology used elsewhere in the document?

Reply:
It's used in the document without definition, so unless you want to replace the current use to something clearer, we should add a definition. There's a "standard CoRIM extension" later referencing CoTS, but that seems like a different notion of extension than the map codepoint extensions.

Reply:
I would expect that any number in IANA registries would link to the standard document that introduced it and gives it meaning, which is why I just used the IANA registry. That's the one authority that can assign numbers that are used in these extension points, no?

Reply:
I think "standard CoRIM type" is referring to the codepoints within `measurement-values-map` but this isn't very clear from the text.

I wonder whether it would be better to say something along the lines of "a profile SHALL not redefine anything in this document, but it can add to it", which is what I think the current text is trying to say.